src/cljs/harja/tiedot/urakka/pot2/materiaalikirjasto.cljs

```diff
@@ -164,8 +164,43 @@
                      (lisa-aineisiin-pitoisuus rivi aineet))))
     aineet))
 
+(def uuden-materiaalin-nimen-vihje "Nimi muodostetaan automaattisesti lomakkeeseen täytettyjen tietojen perusteella")
+
 (def uusi-massa-map
-  {:harja.domain.pot2/sideaineet {:lopputuote {:valittu? true}}})
+  {::pot2-domain/massan-nimi uuden-materiaalin-nimen-vihje
+   :harja.domain.pot2/sideaineet {:lopputuote {:valittu? true}}})
+
+(def uusi-murske-map
+  {::pot2-domain/murskeen-nimi uuden-materiaalin-nimen-vihje})
+
+(defn- materiaalin-nimen-komp [{:keys [ydin tarkennukset fmt toiminto-fn]}]
+  (if (= :komponentti fmt)
```

Comment:
Tähän ei haittaisi, vaikka pikku kommentilla selventäisi tämän toimintaa. Erityisesti tuo yksi iffi jossa tulkitaan funkkaria ja ilmeiseti dom elementtien läsnäoloa? toiminta-fn :div :span.
Jää ainakin mulle vähän arvailujen varaan, että miten se toimii tai miksi.

Reply:
Tässä siis annetaan toiminto-fn, jos halutaan että materiaalin nimestä tulee klikattava. Tällöin elementti wrapätään divin sisään, mutten spanin 

(if toiminto-fn :div :span) 

katonpa jos keksisin miten tuota olennaisilta osilta dokumentoisi. kyseessä kuitenkin vahvasti pot2-spesfinen funktio, eli tästä ei kovin yleiskäyttöistä saa edes dokumentoimalla.